internal/cli/iam/organizations/apikeys/accesslists/delete.go

```diff
@@ -54,11 +55,13 @@ func DeleteBuilder() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "delete <ID>",
 		Aliases: []string{"rm"},
-		Short:   "Delete an IP access list from your API Key.",
+		Short:   "Remove the specified IP access list entry from your API Key.",
 		Args:    require.ExactArgs(1),
 		Annotations: map[string]string{
-			"IDDesc": "Access list identifier.",
+			"IDDesc": "IP or CIDR address that you want to remove from the access list. If the entry includes a subnet mask, such as 192.0.2.0/24, use the URL-encoded value %2F for the forward slash /.",
 		},
+		Example: fmt.Sprintf(`  # Remove the IP address 192.0.2.0 from the access list for the API key with the ID 5f24084d8dbffa3ad3f21234 in the organization with the ID 5a1b39eec902201990f12345:
```

Comment:
Should we also include an example that shows usage of the URL-encoded value for the forward slash?

Reply:
Unfortunately, I tried to do that and it caused an error as it interpreted the % as part of the code. But if engineering knows how I might include it without causing issues I can do that